database/dummyData/userEventData.js

```diff
@@ -84,29 +94,19 @@ const generateMovieProfile = () => {
 };
 
 const generateMovieObj = () => {
-  const rec = Math.round(Math.random()) === 0;
   const movieObj = {
     id: Math.floor(Math.random() * 300000),
     profile: generateMovieProfile(),
-    isRec: rec,
+    isRec: Math.round(Math.random()) === 0,
```

Comment:
This is clever! Maybe instead of the '===' operator, though, you could use a bang (or double bang): e.g. !Math.round(Math.random())